docs/integrators/deep-history-squad.md

```diff
@@ -0,0 +1,155 @@
+---
+id: deep-history-squad
+title: Deep History Squad
```

Comment:
reference this page in the overview page of the `integrators`

Reply:
Added.